lib/ui/semantics/semantics_update_builder.h

```diff
@@ -43,10 +46,15 @@ class SemanticsUpdateBuilder
                   double elevation,
                   double thickness,
                   std::string label,
-                  std::string hint,
+                  std::vector<std::vector<std::any>> labelAttributes,
```

Comment:
Why are we using std::any? Can we avoid it?

Reply:
For example, could we juse use some class(es) instead of vectors of vectors of anything?

Reply:
Is there a way to serialize a custom dart class? I couldn't figure out a way to do that with the dart api

Reply:
We do it by hand in a few places - for example https://github.com/flutter/engine/blob/9c793f1c6b90acd5536d98fd10d2bfa10f84d9c4/lib/ui/painting.dart#L1104-L1116

Reply:
Sounds good